core/assets/jsons/global.json

```diff
@@ -126,7 +126,7 @@
       "density": 1.0,
       "friction": 0.0,
       "restitution": 0.0,
-      "walkspeed": 	  4.0,
+      "walkspeed": 	  2.4,
```

Comment:
The ghost is too slow now. If I managed to get ahead of the ghost, I can literally walk at any speed I want (even sneak a little) and not feel threatened. In short, it almost nerfs the ghost on maze levels. I don't think it should be as fast as before though. I think maybe a speed of 3.0 may suffice.